### DISCUSSION TITLE
Is there a way to rename a column in an expression?

### DISCUSSION THREAD

**Discussion by charlieholtz72** [perspective-dev/perspective#1496] [Q&A] [answered]
For example, let's say within an expression I want to calculate MV as `price * num_shares`. Is there a way to add that column but renamed as `MV`? 

Ideally I'd like to do something like this: `'price' * 'num_shares' as 'MV'`

**Comment by dhirschfeld** [comment, marked as answer]
The new way is to use a comment to denote the column name: https://github.com/finos/perspective/issues/1493#issuecomment-887157801
